UEBPrep/app/controllers/users_controller.rb

```diff
@@ -5,7 +5,7 @@ def index
   end
 
   def show
-    @user = User.friendly.find(user_params[:id])
```

Comment:
Make's show secure-I can't see a viable use case where we would need to direct a signed in user to another user's account page.